controller/brandController.js

```diff
@@ -0,0 +1,112 @@
+const brandService = require("../services/brandService");
+
+//전체 브랜드 조회
+const getAllBrands = async (req, res) => {
+  try {
+    //관리자 여부 조회
+    const token = req.headers.authorization.substr(7);
+    const verifiedToken = await middleJwt.verifyToken(token);
+
+    if (verifiedToken.status != 1) {
+      return res.status(404).json({ message: "관리자 권한이 없습니다" });
+    } //
```

Comment:
```suggestion

    if (verifiedToken.status != 1) {
      const error = new Error("관리자 권한이 없습니다");
      error.statusCode = 403
      throw error
    }
```
- 에러를 리턴하는 대신, throw를 이용해서 try-catch문과 함께 적용해 보세요 :)
- 로그인은 했는데, 관리자 권한이 없는 '권한 에러'의 경우 status code 403이 일반적으로 이용됩니다